packages/dev/inspector/src/components/actionTabs/tabs/propertyGrids/materials/textures/textureEditorComponent.tsx

```diff
@@ -200,7 +200,7 @@ export class TextureEditorComponent extends React.Component<ITextureEditorCompon
 
     textureDidUpdate() {
         if (this._timer != null) {
-            clearTimeout(this._timer);
+            window.clearTimeout(this._timer);

```

Comment:
Here I use `window.clearTimeout` instead of the global `clearTimeout` for consistency, as we call `window.setTimeout` 2 lines below.

Reply:
why not use the global setTimeout and clearTimeout?

Reply:
No good reason, I can change that

Reply:
Actually in this case I can use window's function because we are in a React component.

In order to use global `setTimeout`/`clearTimeout `, I'd have to replace 
```
let throttleTimerId = -1
```
with 
```
let throttleTimerId : ReturnType<typeof setTimeout> | undefined = undefined;
```

and then I could write 
```
            clearTimeout(throttleTimerId);
            throttleTimerId = setTimeout(() => {
                this.raiseOnPropertyChanged(value, store);
            }, this.props.throttlePropertyChangedNotificationDelay ?? 200);
```

Reply:
Shall I do it anyway ?

Reply:
nope, I totally missed the fact that this is a tsx and not in core. keep it this way :-)